bindings/rust/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "gtuber"
+version = "0.0.1"
+edition = "2018"
+description = "Rust bindings for gtuber"
+license = "LGPL-3.0-or-later"
```

Comment:
2.1?

Reply:
I just remember talking about lgpl for gtuber in #clapper. Didn't remember the version, so I just changed the A from my template to L :D

should be fixed now